### PR TITLE
Integrations landing page construction

### PR DESCRIPTION
**Pull request by DhaaraniCIT**

### CODE REVIEW

**Comment by DhaaraniCIT** [comment]
Landing page
Screenshot 2023-10-20 at 12 24 53 PM

File: src/app/integrations/landing/landing.component.html [hunk 1]
```diff
@@ -45,12 +45,20 @@
...
+                            Sage Intacct
+                            <svg-icon-sprite *ngIf="!exposeIntacctNewApp" src="open-in-new-tab" width="14px" height="14px" class="landing--accounting-app-open"></svg-icon-sprite>
+                        </span>
+                    </div>
+                    <!-- <div class="landing--accounting-app tw-mr-20-px tw-mt-20-px" (click)="openInAppIntegration(InAppIntegration.SAGE300)">
+                        <img src="assets/logos/sage300-logo.svg" width="110px" height="60px" />
+                        <span class="landing--accounting-app-name">
+                            Sage 300 CRE
+                        </span>
+                    </div> -->
```
Comment (ashwin1111) [inline src/app/integrations/landing/landing.component.html:61]: I have a feeling this should be added at last, whenever we uncomment let's move it to last

**Review by ashwin1111 — APPROVED** [review]

File: src/app/integrations/landing/landing.component.html (hunk 1 above)
Comment (DhaaraniCIT) [reply src/app/integrations/landing/landing.component.html:61]: this should come after qbd right

Comment (ashwin1111) [reply src/app/integrations/landing/landing.component.html:61]: yes

**Pull request merged**